src/components/ScriptEditor.tsx

```diff
@@ -209,6 +220,24 @@ class ScriptEditor extends React.Component<Props, State> {
     this.setState({ editorState: newEditorState }, this.focus);
   };
 
+  renderAttachmentWarning() {
+    if (!this.state.validAttachment) {
+      return (
+        <div style={{ color: baseTheme.colors.red }}>
+          WARNING! The media attachment URL has type text/html which is not a
```

Comment:
So often when trying to access a URL that might not be valid, or isn't supported might report an issue because of CORS. My suggestion would be obfuscate some of the details, and update the documentation to provide a list of supported types, something along the line of:

"WARNING! The media attachment URL is of an unsupported type MMS type, Please check the URL of the attachment or see Including an Image in a Message for supported attachment types"